### ISSUE TITLE
Nakama 3.0 stopped response on RPC after 32 requests.

### ISSUE THREAD

**Issue by cuongnmx** [heroiclabs/nakama#538]
## Description
Nakama 3.0 stopped response on RPC after 32 requests.
If I run "nakama  --runtime.js_max_count 40" then server will timeout after 40 requests.

## Steps to Reproduce
1. Start nakama
2. Send request from client continuously

server code:

```
const rpcIdTest = 'test_rpc';
let InitModule: nkruntime.InitModule =
    function (ctx: nkruntime.Context, logger: nkruntime.Logger, nk: nkruntime.Nakama, initializer: nkruntime.Initializer) {
        let server = new GameServerCore();
        initializer.registerRpc(rpcIdTest, server.testRPC);
    }

let a = 1;

class GameServerCore {
    testRPC(context: nkruntime.Context, logger: nkruntime.Logger, nk: nkruntime.Nakama, request: string) {
        logger.info("count = " + a++);
        return JSON.stringify({ data: a});
    }
}
```

## Expected Result
Server response on all requests.

## Actual Result
Nakama stopped response after 32 requests, client received "Request timed out".

## Context
Cocos 2.4.3

## Your Environment
- Nakama: 3.0
- Database: CCL v20.1.8
- Environment name and version:
- Operating System and version: CentOS 8

**Comment by novabyte** [comment]
Thank you for the issue report @cuongnmx we'll have a look and make any fixes required in time for our next patch release. 🙏

**Comment by sesposito** [comment]
@cuongnmx this is fixed in #532 so I'll close this issue. Thanks for reporting it.
